darts/models/forecasting/forecasting_model.py

```diff
@@ -1085,6 +1085,14 @@ def load(path: Union[str, BinaryIO]) -> "ForecastingModel":
 
         return model
 
+    def __getstate__(self):
```

Comment:
Is it needed to define these methods here? They'll anyway be overwritten in TFM no?

Reply:
No, not needed. I thought it would make it a bit cleaner but I will remove it :)